service/src/main/java/tennisclub/service/LessonService.java

```diff
@@ -0,0 +1,19 @@
+package tennisclub.service;
+
+import tennisclub.entity.Lesson;
+import tennisclub.entity.User;
+import tennisclub.entity.enums.Level;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
```

Comment:
In the reference eshop project they used the @service annotation also for the interface, I guess we should add it here as well.